.github/workflows/02-pr-checks.yml

```
@@ -5,18 +5,11 @@ on:
       - main
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ghcr.io/volkertb/debian-djgpp:v0.2
```

`runs-on` modifies the runner, you want to switch to a different Docker image:

```suggestion
    runs-on: ubuntu-latest
    container:
      image: ghcr.io/volkertb/debian-djgpp:v0.2
```

---

Thanks! Sorry I couldn't fix this earlier. I was at work during the day. :sweat_smile: